MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryEditSaveImageDetailsTest.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMediaGalleryEditSaveImageDetailsTest">
+        <annotations>
+            <features value="MediaGallery"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/724"/>
+            <title value="User edits image meta data in media gallery"/>
+            <stories value="[Story # 38] User views basic image attributes in Media Gallery"/>
+            <testCaseId value=""/>
```

Comment:
@chalov-anton , please provide the test case id (link) that should be used for this MFTF test

Reply:
> @chalov-anton , please provide the test case id (link) that should be used for this MFTF test

https://studio.cucumber.io/projects/131313/test-plan/folders/1320712/scenarios/3961351
The scenario is _In Progress_ state